gpu_ocean/SWESimulators/WindForcingEnsemble.py

```diff
@@ -42,16 +43,21 @@ class WindForcingEnsemble(BaseOceanStateEnsemble.BaseOceanStateEnsemble):
     def init(self, driftersPerOceanModel=1):
         self.windSpeed = 2.0
         self.directions = np.random.rand(self.numParticles + 1)*360
+        self.windX, self.windY = self.XandYfromDirections(self.directions)
         #print "Directions: ", self.directions
         self.driftersPerOceanModel = driftersPerOceanModel
         
+        self.windT = np.zeros((1), dtype=np.float32)
+        
         for i in range(self.numParticles+1):
-            wind = Common.WindStressParams(type=50, 
-                                           wind_speed=self.windSpeed,
-                                           wind_direction=self.directions[i])
-
             
-            self.particles[i] = CDKLM16.CDKLM16(self.cl_ctx, \
+            wX = [self.windX[i]*np.ones((2,2), dtype=np.float32)]
+            wY = [self.windY[i]*np.ones((2,2), dtype=np.float32)]
+            
+            wind = WindStress.WindStress(self.windT, wX, wY)
```

Comment:
Wow, cool!

Reply:
Not cool, I would say :P The better solution would be to more flexible in the WindStress constructor. If X and Y was given by a list with a single float only (and not a list of np.arrays), the WindStress should create the 2x2 field with the constant value for you.

Reply:
In a related note, I believe there's a bug in the wind stress... See: https://trello.com/c/jKzWJkIW